In [1]:
%matplotlib

Using matplotlib backend: Qt5Agg


In [2]:
import optuna

import os
import sys
import math
import itertools
import numpy as np
import pandas as pd
from scipy.special import perm, comb

from google.cloud import bigquery

In [3]:
client = bigquery.Client()

In [4]:
sql = """
select
  race_info_id,
  race_date,
  place,
  race_num,
  umakazu
from
  [dataset].[table]
where
  race_date < date('2019-06-21')
group by
  race_info_id,
  race_date,
  place,
  race_num,
  umakazu
order by
  race_info_id
"""

In [5]:
df = client.query(sql).to_dataframe()
df.head()

race_info_id   race_date place  race_num  umakazu
0        192991  2019-01-01    川崎         1        7
1        192992  2019-01-01    川崎         2       10
2        192993  2019-01-01    川崎         3       11
3        192994  2019-01-01    川崎         4       10
4        192995  2019-01-01    川崎         5       10

In [6]:
results = client.query("""
select
  r.*  
from (
  select distinct race_info_id from
    [dataset].[table_run]
  where
    race_date < date('2019-06-21')
) as x
  inner join
    [dataset].[table_result] as r
    on r.race_info_id = x.race_info_id
order by
  r.race_info_id
""").to_dataframe()

In [7]:
scores = client.query("""
select
  x.race_info_id,
  x.uma_id,
  x.umaban,
  IF (x.umakazu < y.max_umaban, y.max_umaban, x.umakazu) as umakazu,
  r.score,
  r.value
from (
  select
      race_info_id,
      uma_id,
      umaban,
      umakazu
  from
    [dataset].[table_run] as kr
  where
    race_date < date('2019-06-21')
) as x
  inner join (
    select
      cast(uma_id as INT64) as uma_id,
      cast(race_info_id as INT64) as race_info_id,
      avg(predicted.tables.score) as score,
      predicted.tables.value as value
    from
      [predicted_dataset].predictions as p
      CROSS JOIN UNNEST(p.predicted_is_profit) as predicted
    where
      predicted.tables.value = '1'
    group by
      p.uma_id,
      p.race_info_id,
      predicted.tables.value
  ) as r
    on r.race_info_id = x.race_info_id
       and r.uma_id = x.uma_id
  inner join (
    select
      race_info_id,
      max(umaban) as max_umaban
    from
      [dataset].[table_run]
    group by
      race_info_id
  ) as y
    on y.race_info_id = x.race_info_id
order by
  r.race_info_id
""").to_dataframe()
scores.head()

race_info_id  uma_id umaban umakazu     score value
0        192991   14036      2       8  0.529314     1
1        192991   13703      3       8  0.306875     1
2        192991   12891      4       8  0.569858     1
3        192991   13319      5       8  0.370065     1
4        192991   59473      6       8  0.402599     1

In [8]:
def umaban2waku(umakazu):
    m = 8

    a = m * 2 - umakazu
    n = 0
    step = 1
    ws = {}
    for i in range(umakazu):
        if i >= a:
            if step == 0:
                step = 1
            else:
                n += 1
                step = 0
        else:
            n += 1
        ws[i+1] = n
    return ws

In [9]:
def tanpuku(score_df, select_num):
    return [[int(x)] for x in list(score_df[:select_num].kumiban)]

In [10]:
def uma_ren(score_df, sub, select_num, ticket_unit):
    kumibans = list(score_df.kumiban)

    pairs = []
    if sub == 'box':
        pairs = list(itertools.combinations(kumibans[:select_num], ticket_unit))
    elif sub == 'flow':
        jiku = kumibans[0]
        pairs = [ (jiku, x) for x in kumibans[1:] ]
    elif sub == 'form2':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]

        pairs.append((jiku1, jiku2))
        pairs.append((jiku1, jiku2))
        pairs += [ (jiku1, x) for x in kumibans[2:] ]
        pairs += [ (jiku2, x) for x in kumibans[2:] ]

    pairs = [sorted(x) for x in pairs ]
    return pairs

In [11]:
def uma_tan(score_df, sub, select_num, ticket_unit):
    kumibans = list(score_df.kumiban)

    pairs = []
    if sub == 'mbox':
        pairs = list(itertools.permutations(kumibans[:select_num], ticket_unit))
    elif sub == 'flow':
        jiku = kumibans[0]
        pairs = [ (jiku, x) for x in kumibans[1:] ]
    elif sub == 'mflow':
        jiku = kumibans[0]
        pairs += [ (jiku, x) for x in kumibans[1:] ]
        pairs += [ (x, jiku) for x in kumibans[1:] ]

    return pairs

In [12]:
def sanren_puku(score_df, sub, select_num, ticket_unit):
    kumibans = list(score_df.kumiban)

    pairs = []
    if sub == 'box':
        pairs = list(itertools.combinations(kumibans[:select_num], ticket_unit))
    elif sub == 'flow1':
        jiku = kumibans[0]
        others = list(itertools.combinations(kumibans[1:select_num], 2))
        pairs = [ (jiku, x[0], x[1]) for x in others ]
    elif sub == 'flow2':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        pairs += [ (jiku1, jiku2, x) for x in kumibans[2:] ]
    elif sub == 'form21':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        others1 = list(itertools.combinations(kumibans[1:select_num], 2))
        others2 = list(itertools.combinations(kumibans[2:select_num], 2))
        pairs += [ (jiku1, x[0], x[1]) for x in others1 ]
        pairs += [ (jiku2, x[0], x[1]) for x in others2 ]
    elif sub == 'form22':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        jiku3 = kumibans[2]
        pairs += [ (jiku1, jiku2, x) for x in kumibans[2:] ]
        pairs += [ (jiku2, jiku3, x) for x in kumibans[3:] ]  # jiku1とjiku3 からの流しが無い...！

    pairs = [sorted(x) for x in pairs ]
    return pairs

In [13]:
def sanren_tan(score_df, sub, select_num, ticket_unit):
    kumibans = list(score_df.kumiban)

    pairs = []
    if sub == 'mbox':
        pairs = list(itertools.permutations(kumibans[:select_num], ticket_unit))
    elif sub == 'tflow1':
        jiku = kumibans[0]
        others = list(itertools.permutations(kumibans[1:select_num], 2))
        pairs = [ (jiku, x[0], x[1]) for x in others ]
    elif sub == 'tflow2':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        pairs = [ (jiku1, jiku2, x) for x in kumibans[2:] ]
    elif sub == 'mflow1':
        jiku = kumibans[0]
        for i in range(len(kumibans) - 2):
            pairs += list(itertools.permutations([jiku, kumibans[i+1], kumibans[i+2]]))
    elif sub == 'mflow2':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        for i in range(len(kumibans) - 2):
            pairs += list(itertools.permutations([jiku1, jiku2, kumibans[i+2]]))
    elif sub == 'form2t1':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        others1 = list(itertools.permutations(kumibans[1:select_num], 2))
        others2 = list(itertools.permutations(kumibans[2:select_num], 2))
        pairs += [ (jiku1, x[0], x[1]) for x in others1 ]
        pairs += [ (jiku2, x[0], x[1]) for x in others2 ]
    elif sub == 'form2t2':
        jiku1 = kumibans[0]
        jiku2 = kumibans[1]
        jiku3 = kumibans[2]
        pairs += [ (jiku1, jiku2, x) for x in kumibans[2:] ]
        pairs += [ (jiku2, jiku3, x) for x in kumibans[3:] ]  # jiku1とjiku3 からの流しが無い...！

    return pairs

In [14]:
def buy_tickets(pairs, ratio, max_w):
    ws = 0
    boughts = []

    for idx in range(len(pairs)):
        if ws >= max_w:
            break

        bet = math.floor(float(max_w) * ratio[idx])
        ws += bet

        kumiban = ''
        if len(pairs[idx]) == 2:
            kumiban = "{0}-{1}".format(pairs[idx][0], pairs[idx][1])
        elif len(pairs[idx]) == 3:
            kumiban = "{0}-{1}-{2}".format(pairs[idx][0], pairs[idx][1], pairs[idx][2])
        else:
            kumiban = "{0}".format(pairs[idx][0])

        boughts.append({'kumiban': kumiban, 'bet': bet})

    idx = 0
    for j in range(max_w - ws):
        boughts[idx]['bet'] += 1
        idx += 1
        if idx >= len(boughts):
            idx = 0

    return boughts

## 枠番の追加

※ inken の取得したデータに枠番が入ってなかったので馬番から計算しました.

In [15]:
def uma2waku(umakazu, umaban):
    d = umaban2waku(int(umakazu))
    v = None
    try:
        v = d[int(umaban)]
    except:
        print(umakazu, umaban)
    return d[int(umaban)]

scores['waku'] = scores.apply(lambda x: uma2waku(x['umakazu'], x['umaban']), axis=1)
scores.head()

race_info_id  uma_id umaban umakazu     score value  waku
0        192991   14036      2       8  0.529314     1     2
1        192991   13703      3       8  0.306875     1     3
2        192991   12891      4       8  0.569858     1     4
3        192991   13319      5       8  0.370065     1     5
4        192991   59473      6       8  0.402599     1     6

## 式別毎購入可能点数の計算

予算範囲内で購入可能な点数を式別ごとに算出しておきます.

In [16]:
def count_marks(sub, select_num, uma_num):
    if sub == 'none':
        return select_num
    elif sub == 'box':
        return comb(select_num, uma_num, exact=True)
    elif sub == 'mbox':   # 単裏表
        return perm(select_num, uma_num, exact=True)
    elif sub == 'flow':   # 馬連などの流し
        return select_num - 1
    elif sub == 'mflow':  # 馬単マルチ
        return (select_num - 1) * 2
    elif sub == 'form2':  # 2頭フォーメーション
        return (select_num - 1) * 2 
    elif sub == 'flow1':  # 三連複軸1頭
        return comb(select_num - 1, 2, exact=True)
    elif sub == 'flow2':  # 三連複軸2頭
        return select_num - 2
    elif sub == 'form21':  # 三連複軸1頭-2頭フォーメーション
        return comb(select_num - 1, 2, exact=True) + comb(select_num - 2, 2, exact=True)
    elif sub == 'form22':  # 三連複軸2頭-2頭フォーメーション
        return (select_num - 2) + (select_num - 3)
    elif sub == 'tflow1':  # 三連単軸1頭固定
        return perm(select_num - 1, 2, exact=True)
    elif sub == 'tflow2':  # 三連単軸2頭固定
        return select_num - 2
    elif sub == 'mflow1':  # 三連単軸1頭マルチ
        return perm(select_num - 1, 2, exact=True) * 3
    elif sub == 'mflow2':  # 三連単軸2頭マルチ
        return (select_num - 2) * 6
    elif sub == 'form2t1': # 三連単軸1頭-2頭フォーメーション
        return perm(select_num - 1, 2, exact=True) + perm(select_num - 2, 2, exact=True)
    elif sub == 'form2t2':  # 三連単軸2頭-2頭フォーメーション
        return (select_num - 2) + (select_num - 3)

In [17]:
def search_threshold(sub, max_w, s, m):
    res = s
    for n in range(s, m + 1):
        mx = count_marks(sub, n, s)
        if mx > max_w:
            break
        res = n
    return res

In [18]:
MAX_PAYMENT = 2100  # 1レース当たりの購入上限を¥2,100 としています
MAX_UMAKAZU = 16
MAX_WAKUKAZU = 8
unit = 100    # 購入券の最小金額
max_w = int(MAX_PAYMENT / unit)

## 設定した金額で買える最大組数を探します
s_config = {}


# 単勝, 複勝
s_config['none'] = {
    '単勝': { 'low': 1, 'high': min([MAX_UMAKAZU, max_w]) },
    '複勝': { 'low': 1, 'high': min([MAX_UMAKAZU, max_w]) },
}

#------------------------------------------------------
# box
sub = 'box'
uma_num = 2

mx = search_threshold(sub, max_w, uma_num, MAX_UMAKAZU)
s_config[sub] = {
    '普通馬複': { 'low': uma_num, 'high': mx },
    'ワイド': { 'low': uma_num, 'high': mx },    
    '枠複': { 'low': uma_num, 'high': search_threshold(sub, max_w, uma_num, MAX_WAKUKAZU) },
    '三連複': { 'low': 3, 'high': search_threshold(sub, max_w, 3, MAX_UMAKAZU) },
}

#------------------------------------------------------
# mbox 単裏表
sub = 'mbox'
uma_num = 2

s_config[sub] = {
    '馬単': { 'low': uma_num, 'high': search_threshold(sub, max_w, uma_num, MAX_UMAKAZU) },
    '枠単': { 'low': uma_num, 'high': search_threshold(sub, max_w, uma_num, MAX_WAKUKAZU) },
    '三連単': { 'low': 3, 'high': search_threshold(sub, max_w, 3, MAX_UMAKAZU) },
}

#------------------------------------------------------
# flow 流し
sub = 'flow'
uma_num = 2

mx = min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU])
mxw = min([search_threshold(sub, max_w, uma_num, MAX_WAKUKAZU), MAX_WAKUKAZU])
s_config[sub] = {
    '普通馬複': { 'low': uma_num, 'high': mx },
    'ワイド': { 'low': uma_num, 'high': mx },
    '馬単': { 'low': uma_num, 'high': mx },
    '枠複': { 'low': uma_num, 'high': mxw },
    '枠単': { 'low': uma_num, 'high': mxw },
}

#------------------------------------------------------
# mflow 馬単マルチ
sub = 'mflow'
uma_num = 2

s_config[sub] = {
    '枠単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_WAKUKAZU), MAX_WAKUKAZU]) },
    '馬単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}

#------------------------------------------------------
# form2 フォーメーション
sub = 'form2'
uma_num = 2

mx = min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU])
mxw = min([search_threshold(sub, max_w, uma_num, MAX_WAKUKAZU), MAX_WAKUKAZU])
s_config[sub] = {
    '普通馬複': { 'low': 4, 'high': mx },
    'ワイド': { 'low': 4, 'high': mx },
    '枠複': { 'low': 4, 'high': mxw },
}


#------------------------------------------------------
# 三連複 flow1, flow2
uma_num = 3

sub = 'flow1'  # 三連複軸1頭
s_config[sub] = {
    '三連複': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}
sub = 'flow2'  # 三連複軸2頭
s_config[sub] = {
    '三連複': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },    
}

#------------------------------------------------------
# form21, form22 三連複フォーメーション
uma_num = 3

sub = 'form21'
s_config[sub] = {
    '三連複': { 'low': 4, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}

sub = 'form22'
s_config[sub] = {
    '三連複': { 'low': 4, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}

#------------------------------------------------------
# 三連単 tflow1, tflow2, mflow1, mflow2
uma_num = 3

sub = 'tflow1'  # 三連単軸1頭固定
s_config[sub] = {
    '三連単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}
sub = 'tflow2'  # 三連単軸2頭固定
s_config[sub] = {
    '三連単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}
sub = 'mflow1'  # 三連単軸1頭マルチ
s_config[sub] = {
    '三連単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}
sub = 'mflow2'  # 三連単軸2頭マルチ
s_config[sub] = {
    '三連単': { 'low': uma_num, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}

#------------------------------------------------------
# form2t1, form2t2 三連単フォーメーション
uma_num = 3

sub = 'form2t1'
s_config[sub] = {
    '三連単': { 'low': 4, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}

sub = 'form2t2'
s_config[sub] = {
    '三連単': { 'low': 4, 'high': min([search_threshold(sub, max_w, uma_num, MAX_UMAKAZU), MAX_UMAKAZU]) },
}


In [19]:
types = {}
for k, v in s_config.items():
    for shiki, m in v.items():
        if shiki not in types:
            types[shiki] = {k: m}
        else:
            types[shiki][k] = m
types

{'単勝': {'none': {'low': 1, 'high': 16}},
 '複勝': {'none': {'low': 1, 'high': 16}},
 '普通馬複': {'box': {'low': 2, 'high': 7},
  'flow': {'low': 2, 'high': 16},
  'form2': {'low': 4, 'high': 11}},
 'ワイド': {'box': {'low': 2, 'high': 7},
  'flow': {'low': 2, 'high': 16},
  'form2': {'low': 4, 'high': 11}},
 '枠複': {'box': {'low': 2, 'high': 7},
  'flow': {'low': 2, 'high': 8},
  'form2': {'low': 4, 'high': 8}},
 '三連複': {'box': {'low': 3, 'high': 6},
  'flow1': {'low': 3, 'high': 8},
  'flow2': {'low': 3, 'high': 16},
  'form21': {'low': 4, 'high': 6},
  'form22': {'low': 4, 'high': 13}},
 '馬単': {'mbox': {'low': 2, 'high': 5},
  'flow': {'low': 2, 'high': 16},
  'mflow': {'low': 2, 'high': 11}},
 '枠単': {'mbox': {'low': 2, 'high': 5},
  'flow': {'low': 2, 'high': 8},
  'mflow': {'low': 2, 'high': 8}},
 '三連単': {'mbox': {'low': 3, 'high': 3},
  'tflow1': {'low': 3, 'high': 6},
  'tflow2': {'low': 3, 'high': 16},
  'mflow1': {'low': 3, 'high': 4},
  'mflow2': {'low': 3, 'high': 5},
  'form2t1': {'l

In [74]:
UMAKAZU = 16  # 5〜16

targets = df[df['umakazu'] == UMAKAZU]
targets.head()

race_info_id   race_date place  race_num  umakazu
155        193733  2019-01-21    大井         3       16
159        193737  2019-01-21    大井         7       16
160        193738  2019-01-21    大井         8       16
164        193742  2019-01-21    大井        12       16
166        193765  2019-01-22    大井         2       16

In [75]:
if UMAKAZU > 8:
    result_types = ['単勝', '複勝', '枠複', '枠単', '普通馬複', '馬単', 'ワイド', '三連複', '三連単']
else:
    result_types = ['単勝', '複勝', '普通馬複', '馬単', 'ワイド', '三連複', '三連単']

shiki_list = []
for k, v in types.items():
    for s in v.keys():
        shiki_list.append("{0}-{1}".format(k, s))
print("種類数", len(shiki_list))
shiki_list

種類数 29


['単勝-none',
 '複勝-none',
 '普通馬複-box',
 '普通馬複-flow',
 '普通馬複-form2',
 'ワイド-box',
 'ワイド-flow',
 'ワイド-form2',
 '枠複-box',
 '枠複-flow',
 '枠複-form2',
 '三連複-box',
 '三連複-flow1',
 '三連複-flow2',
 '三連複-form21',
 '三連複-form22',
 '馬単-mbox',
 '馬単-flow',
 '馬単-mflow',
 '枠単-mbox',
 '枠単-flow',
 '枠単-mflow',
 '三連単-mbox',
 '三連単-tflow1',
 '三連単-tflow2',
 '三連単-mflow1',
 '三連単-mflow2',
 '三連単-form2t1',
 '三連単-form2t2']

In [76]:
def objective(trial):

    targets = df[df['umakazu'] == UMAKAZU]

    # shiki: 式別 '単勝', '複勝', '枠複', '枠単', '普通馬複', '馬単', 'ワイド', '三連複', '三連単'
    shiki_sub = trial.suggest_categorical('shiki', shiki_list)

    # pattern: BOX, 流し, マルチなど
#     sub = trial.suggest_categorical('sub', list(types[shiki].keys()))  # カテゴリー値は動的変数にできなかった(´・ω・｀)ｼｮﾎﾞｰﾝ
    shiki, sub = shiki_sub.split('-')


    # ticket_unit: 式別の単位数
    if shiki in ['単勝', '複勝']:
        ticket_unit = 1
    elif shiki in ['三連複', '三連単']:
        ticket_unit = 3
    else:
        ticket_unit = 2


    # pattern: 均等買い or 重み買い
    if sub in ['none', 'flow', 'flow2', 'tflow2']:
        pattern = trial.suggest_categorical('pattern', ['eq', 'order_s', 'order_o'])
    else:
        pattern = 'eq'


    # select_num: 選択馬数
    select_num = trial.suggest_int('select_num', types[shiki][sub]['low'], min([types[shiki][sub]['high'], UMAKAZU]))


    print(shiki, sub, select_num, pattern, trial.number)

    # この値を最大化します
    profit = 0

    for i in list(targets['race_info_id']):
        ss = pd.DataFrame(scores[scores['race_info_id'] == i].sort_values('score', ascending=False))

        if shiki in ['枠複', '枠単']:
            ss['kumiban'] = ss['waku']
        else:
            ss['kumiban'] = ss['umaban']

        start_index = 0
        if sub in ['flow2', 'tflow2']:
            start_index = 2
        elif sub in ['flow']:
            start_index = 1

        rdf = pd.DataFrame(ss)
        rdf = rdf[start_index:select_num]
        rdf = rdf.reset_index()

        marks = count_marks(sub, select_num, ticket_unit)
        if pattern == 'eq':
            ratio = [ (1.0/float(marks)) for _ in range(marks)]
        elif pattern == 'order_s':
            ratio = list(rdf.loc[:, ['score']].apply(lambda x:x/sum(x))['score'])
        elif pattern == 'order_o':
            rdf['order'] = rdf.index * -1 + select_num
            ratio = list(rdf.loc[:, ['order']].apply(lambda x:x/sum(x))['order'])

        ss = ss[:select_num]
        ss = ss.reset_index()


        pairs = []
        if shiki == '単勝' or shiki == '複勝':
            pairs = tanpuku(ss, select_num)
        elif shiki == '普通馬複' or shiki == 'ワイド' or shiki == '枠複':
            pairs = uma_ren(ss, sub, select_num, ticket_unit)
        elif shiki == '馬単' or shiki == '枠単':
            pairs = uma_tan(ss, sub, select_num, ticket_unit)
        elif shiki == '三連複':
            pairs = sanren_puku(ss, sub, select_num, ticket_unit)
        elif shiki == '三連単':
            pairs = sanren_tan(ss, sub, select_num, ticket_unit)


        boughts = buy_tickets(pairs, ratio, max_w)


#         print(boughts)

        # 答え合わせ
        for b in boughts:
            res = results[(results['race_info_id'] == i) & (results['result_type'] == shiki) & (results['kumiban'] == b['kumiban'])]
            loss = int(list(res.price)[0]) if len(res) > 0 else 0.0
#             print(((loss - unit) * b['bet']), b['bet'], (loss - unit))
            profit = profit + ((loss - unit) * b['bet'])

    print("profit", profit)
    return -1.0 * profit

In [77]:
study = optuna.create_study()
study.optimize(objective, n_trials=300)

枠複 form2 5 eq 0
profit -80430.0


[I 2019-06-27 16:07:25,064] Finished trial#0 resulted in value: 80430.0. Current best value is 80430.0 with parameters: {'shiki': '枠複-form2', 'select_num': 5}.


単勝 none 13 eq 1
profit -87630.0


[I 2019-06-27 16:07:37,284] Finished trial#1 resulted in value: 87630.0. Current best value is 80430.0 with parameters: {'shiki': '枠複-form2', 'select_num': 5}.


三連単 mflow2 5 eq 2
profit -90900.0


[I 2019-06-27 16:07:53,556] Finished trial#2 resulted in value: 90900.0. Current best value is 80430.0 with parameters: {'shiki': '枠複-form2', 'select_num': 5}.


三連単 form2t1 5 eq 3
profit -98750.0


[I 2019-06-27 16:08:10,139] Finished trial#3 resulted in value: 98750.0. Current best value is 80430.0 with parameters: {'shiki': '枠複-form2', 'select_num': 5}.


三連単 tflow1 4 eq 4
profit -150870.0


[I 2019-06-27 16:08:16,209] Finished trial#4 resulted in value: 150870.0. Current best value is 80430.0 with parameters: {'shiki': '枠複-form2', 'select_num': 5}.


馬単 mbox 2 eq 5
profit -7110.0


[I 2019-06-27 16:08:18,710] Finished trial#5 resulted in value: 7110.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


普通馬複 form2 9 eq 6
profit -103670.0


[I 2019-06-27 16:08:32,975] Finished trial#6 resulted in value: 103670.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


枠単 mflow 3 eq 7
profit -42850.0


[I 2019-06-27 16:08:37,166] Finished trial#7 resulted in value: 42850.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


枠複 box 7 eq 8
profit -119300.0


[I 2019-06-27 16:08:55,892] Finished trial#8 resulted in value: 119300.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


普通馬複 flow 8 eq 9
profit -52140.0


[I 2019-06-27 16:09:02,415] Finished trial#9 resulted in value: 52140.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


馬単 mbox 2 eq 10
profit -7110.0


[I 2019-06-27 16:09:04,915] Finished trial#10 resulted in value: 7110.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


馬単 flow 16 eq 11
profit -62750.0


[I 2019-06-27 16:09:18,115] Finished trial#11 resulted in value: 62750.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


ワイド box 2 eq 12
profit -51240.0


[I 2019-06-27 16:09:19,847] Finished trial#12 resulted in value: 51240.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


三連単 mflow1 3 eq 13
profit -103560.0


[I 2019-06-27 16:09:25,842] Finished trial#13 resulted in value: 103560.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


ワイド form2 11 eq 14
profit -82750.0


[I 2019-06-27 16:09:43,803] Finished trial#14 resulted in value: 82750.0. Current best value is 7110.0 with parameters: {'shiki': '馬単-mbox', 'select_num': 2}.


三連単 tflow2 15 order_o 15
profit 54550.0


[I 2019-06-27 16:09:56,273] Finished trial#15 resulted in value: -54550.0. Current best value is -54550.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_o', 'select_num': 15}.


三連単 tflow2 16 order_s 16
profit 57030.0


[I 2019-06-27 16:10:10,190] Finished trial#16 resulted in value: -57030.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 tflow2 16 order_o 17
profit 54550.0


[I 2019-06-27 16:10:23,123] Finished trial#17 resulted in value: -54550.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 tflow2 14 eq 18
profit 15620.0


[I 2019-06-27 16:10:34,210] Finished trial#18 resulted in value: -15620.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


枠複 flow 7 order_s 19
profit -58310.0


[I 2019-06-27 16:10:40,677] Finished trial#19 resulted in value: 58310.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 form2t2 12 eq 20
profit -68600.0


[I 2019-06-27 16:10:57,628] Finished trial#20 resulted in value: 68600.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連複 box 6 eq 21
profit -116250.0


[I 2019-06-27 16:11:15,558] Finished trial#21 resulted in value: 116250.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


枠単 flow 8 order_o 22
profit -53830.0


[I 2019-06-27 16:11:22,778] Finished trial#22 resulted in value: 53830.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連複 flow2 16 order_s 23
profit -43990.0


[I 2019-06-27 16:11:35,823] Finished trial#23 resulted in value: 43990.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


普通馬複 box 4 eq 24
profit -105510.0


[I 2019-06-27 16:11:41,740] Finished trial#24 resulted in value: 105510.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連複 flow1 8 eq 25
profit -84790.0


[I 2019-06-27 16:12:01,799] Finished trial#25 resulted in value: 84790.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


複勝 none 10 order_s 26
profit -61910.0


[I 2019-06-27 16:12:11,997] Finished trial#26 resulted in value: 61910.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 tflow2 15 order_s 27
profit 38190.0


[I 2019-06-27 16:12:24,024] Finished trial#27 resulted in value: -38190.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連複 form22 13 eq 28
profit -137870.0


[I 2019-06-27 16:12:42,769] Finished trial#28 resulted in value: 137870.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


ワイド flow 15 order_s 29
profit -64280.0


[I 2019-06-27 16:12:55,781] Finished trial#29 resulted in value: 64280.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 mbox 3 eq 30
profit -103560.0


[I 2019-06-27 16:13:01,635] Finished trial#30 resulted in value: 103560.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


馬単 mflow 11 eq 31
profit -49090.0


[I 2019-06-27 16:13:19,417] Finished trial#31 resulted in value: 49090.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連複 form21 6 eq 32
profit -109530.0


[I 2019-06-27 16:13:34,131] Finished trial#32 resulted in value: 109530.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


枠単 mbox 4 eq 33
profit -98810.0


[I 2019-06-27 16:13:45,151] Finished trial#33 resulted in value: 98810.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


枠複 form2 7 eq 34
profit -90810.0


[I 2019-06-27 16:13:56,303] Finished trial#34 resulted in value: 90810.0. Current best value is -57030.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 16}.


三連単 tflow2 13 order_s 35
profit 82420.0


[I 2019-06-27 16:14:07,102] Finished trial#35 resulted in value: -82420.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 form2t1 5 eq 36
profit -98750.0


[I 2019-06-27 16:14:23,299] Finished trial#36 resulted in value: 98750.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


単勝 none 13 order_s 37
profit -75580.0


[I 2019-06-27 16:14:35,770] Finished trial#37 resulted in value: 75580.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 mflow2 5 eq 38
profit -90900.0


[I 2019-06-27 16:14:52,047] Finished trial#38 resulted in value: 90900.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 tflow1 6 eq 39
profit -162640.0


[I 2019-06-27 16:15:10,226] Finished trial#39 resulted in value: 162640.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


普通馬複 form2 10 eq 40
profit -83070.0


[I 2019-06-27 16:15:27,136] Finished trial#40 resulted in value: 83070.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


枠単 mflow 8 eq 41
profit -76590.0


[I 2019-06-27 16:15:39,863] Finished trial#41 resulted in value: 76590.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


枠複 box 4 eq 42
profit -93270.0


[I 2019-06-27 16:15:45,762] Finished trial#42 resulted in value: 93270.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 tflow2 14 order_s 43
profit 38190.0


[I 2019-06-27 16:15:57,247] Finished trial#43 resulted in value: -38190.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


馬単 flow 14 order_o 44
profit -30910.0


[I 2019-06-27 16:16:09,796] Finished trial#44 resulted in value: 30910.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


普通馬複 flow 12 order_s 45
profit -76360.0


[I 2019-06-27 16:16:20,488] Finished trial#45 resulted in value: 76360.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


ワイド form2 9 eq 46
profit -72210.0


[I 2019-06-27 16:16:34,890] Finished trial#46 resulted in value: 72210.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


ワイド box 3 eq 47
profit -83580.0


[I 2019-06-27 16:16:38,339] Finished trial#47 resulted in value: 83580.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


馬単 mbox 2 eq 48
profit -7110.0


[I 2019-06-27 16:16:40,838] Finished trial#48 resulted in value: 7110.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 mflow1 3 eq 49
profit -103560.0


[I 2019-06-27 16:16:46,907] Finished trial#49 resulted in value: 103560.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 tflow2 13 order_s 50
profit 82420.0


[I 2019-06-27 16:16:57,707] Finished trial#50 resulted in value: -82420.0. Current best value is -82420.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 13}.


三連単 tflow2 12 order_s 51
profit 97600.0


[I 2019-06-27 16:17:08,054] Finished trial#51 resulted in value: -97600.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


枠複 flow 7 order_s 52
profit -58310.0


[I 2019-06-27 16:17:14,595] Finished trial#52 resulted in value: 58310.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連単 form2t2 12 eq 53
profit -68600.0


[I 2019-06-27 16:17:31,860] Finished trial#53 resulted in value: 68600.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連複 box 6 eq 54
profit -116250.0


[I 2019-06-27 16:17:49,809] Finished trial#54 resulted in value: 116250.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


枠単 flow 8 order_s 55
profit -52980.0


[I 2019-06-27 16:17:57,128] Finished trial#55 resulted in value: 52980.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連複 flow2 11 order_s 56
profit -21360.0


[I 2019-06-27 16:18:06,093] Finished trial#56 resulted in value: 21360.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


普通馬複 box 5 eq 57
profit -140220.0


[I 2019-06-27 16:18:15,299] Finished trial#57 resulted in value: 140220.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連複 flow1 7 eq 58
profit -93170.0


[I 2019-06-27 16:18:29,036] Finished trial#58 resulted in value: 93170.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連複 form22 10 eq 59
profit -58290.0


[I 2019-06-27 16:18:42,963] Finished trial#59 resulted in value: 58290.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


ワイド flow 15 order_s 60
profit -64280.0


[I 2019-06-27 16:18:56,209] Finished trial#60 resulted in value: 64280.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


馬単 mflow 9 eq 61
profit -82950.0


[I 2019-06-27 16:19:11,069] Finished trial#61 resulted in value: 82950.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連単 tflow2 12 eq 62
profit 26850.0


[I 2019-06-27 16:19:20,549] Finished trial#62 resulted in value: -26850.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連単 mbox 3 eq 63
profit -103560.0


[I 2019-06-27 16:19:26,648] Finished trial#63 resulted in value: 103560.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連複 form21 4 eq 64
profit 5710.0


[I 2019-06-27 16:19:30,954] Finished trial#64 resulted in value: -5710.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


複勝 none 13 order_o 65
profit -74410.0


[I 2019-06-27 16:19:43,687] Finished trial#65 resulted in value: 74410.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連単 tflow2 13 order_s 66
profit 82420.0


[I 2019-06-27 16:19:54,560] Finished trial#66 resulted in value: -82420.0. Current best value is -97600.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 12}.


三連単 tflow2 11 order_s 67
profit 108830.0


[I 2019-06-27 16:20:03,836] Finished trial#67 resulted in value: -108830.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 2 eq 68
profit -6750.0


[I 2019-06-27 16:20:06,394] Finished trial#68 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 11 eq 69
profit 51730.0


[I 2019-06-27 16:20:15,180] Finished trial#69 resulted in value: -51730.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 14 order_s 70
profit -71000.0


[I 2019-06-27 16:20:28,929] Finished trial#70 resulted in value: 71000.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 7 eq 71
profit -90810.0


[I 2019-06-27 16:20:40,775] Finished trial#71 resulted in value: 90810.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 4 eq 72
profit -46500.0


[I 2019-06-27 16:20:52,126] Finished trial#72 resulted in value: 46500.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 5 eq 73
profit -98750.0


[I 2019-06-27 16:21:08,887] Finished trial#73 resulted in value: 98750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 eq 74
profit 26850.0


[I 2019-06-27 16:21:18,553] Finished trial#74 resulted in value: -26850.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 6 eq 75
profit -162640.0


[I 2019-06-27 16:21:36,773] Finished trial#75 resulted in value: 162640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 76
profit -104490.0


[I 2019-06-27 16:21:55,038] Finished trial#76 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 8 eq 77
profit -76590.0


[I 2019-06-27 16:22:08,253] Finished trial#77 resulted in value: 76590.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 6 eq 78
profit -102870.0


[I 2019-06-27 16:22:22,486] Finished trial#78 resulted in value: 102870.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 order_o 79
profit 68030.0


[I 2019-06-27 16:22:30,772] Finished trial#79 resulted in value: -68030.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 14 eq 80
profit -74920.0


[I 2019-06-27 16:22:42,375] Finished trial#80 resulted in value: 74920.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 3 eq 81
profit -83580.0


[I 2019-06-27 16:22:45,800] Finished trial#81 resulted in value: 83580.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 4 eq 82
profit -148330.0


[I 2019-06-27 16:22:56,738] Finished trial#82 resulted in value: 148330.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 16 order_s 83
profit -33930.0


[I 2019-06-27 16:23:10,395] Finished trial#83 resulted in value: 33930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 2 eq 84
profit -7110.0


[I 2019-06-27 16:23:12,963] Finished trial#84 resulted in value: 7110.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 15 order_s 85
profit 38190.0


[I 2019-06-27 16:23:25,102] Finished trial#85 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 86
profit -82750.0


[I 2019-06-27 16:23:42,838] Finished trial#86 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_o 87
profit -65930.0


[I 2019-06-27 16:23:50,172] Finished trial#87 resulted in value: 65930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 9 eq 88
profit 34210.0


[I 2019-06-27 16:24:02,377] Finished trial#88 resulted in value: -34210.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 6 eq 89
profit -116250.0


[I 2019-06-27 16:24:20,290] Finished trial#89 resulted in value: 116250.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 7 eq 90
profit -88140.0


[I 2019-06-27 16:24:38,958] Finished trial#90 resulted in value: 88140.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 8 eq 91
profit -67080.0


[I 2019-06-27 16:24:45,803] Finished trial#91 resulted in value: 67080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 12 order_s 92
profit -22800.0


[I 2019-06-27 16:24:55,734] Finished trial#92 resulted in value: 22800.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 5 eq 93
profit -116020.0


[I 2019-06-27 16:25:01,839] Finished trial#93 resulted in value: 116020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 13 eq 94
profit -137870.0


[I 2019-06-27 16:25:20,380] Finished trial#94 resulted in value: 137870.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 10 order_o 95
profit -71380.0


[I 2019-06-27 16:25:29,494] Finished trial#95 resulted in value: 71380.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 10 eq 96
profit -32320.0


[I 2019-06-27 16:25:46,370] Finished trial#96 resulted in value: 32320.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 97
profit 38190.0


[I 2019-06-27 16:25:58,349] Finished trial#97 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 98
profit -103560.0


[I 2019-06-27 16:26:04,416] Finished trial#98 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 15 order_s 99
profit -63440.0


[I 2019-06-27 16:26:18,547] Finished trial#99 resulted in value: 63440.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 6 eq 100
profit -109530.0


[I 2019-06-27 16:26:33,179] Finished trial#100 resulted in value: 109530.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 2 eq 101
profit -6750.0


[I 2019-06-27 16:26:35,746] Finished trial#101 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 eq 102
profit 26850.0


[I 2019-06-27 16:26:45,166] Finished trial#102 resulted in value: -26850.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 16 order_s 103
profit -73710.0


[I 2019-06-27 16:27:00,050] Finished trial#103 resulted in value: 73710.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 4 eq 104
profit -142020.0


[I 2019-06-27 16:27:07,970] Finished trial#104 resulted in value: 142020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 5 eq 105
profit -139070.0


[I 2019-06-27 16:27:19,214] Finished trial#105 resulted in value: 139070.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 7 eq 106
profit -90810.0


[I 2019-06-27 16:27:30,716] Finished trial#106 resulted in value: 90810.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 5 eq 107
profit -90900.0


[I 2019-06-27 16:27:46,887] Finished trial#107 resulted in value: 90900.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 108
profit -104490.0


[I 2019-06-27 16:28:04,846] Finished trial#108 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 8 eq 109
profit -76590.0


[I 2019-06-27 16:28:17,619] Finished trial#109 resulted in value: 76590.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_o 110
profit 72290.0


[I 2019-06-27 16:28:28,274] Finished trial#110 resulted in value: -72290.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 7 eq 111
profit -119300.0


[I 2019-06-27 16:28:46,978] Finished trial#111 resulted in value: 119300.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 9 order_s 112
profit -72930.0


[I 2019-06-27 16:28:55,043] Finished trial#112 resulted in value: 72930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 6 eq 113
profit -88650.0


[I 2019-06-27 16:29:08,885] Finished trial#113 resulted in value: 88650.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 114
profit 38190.0


[I 2019-06-27 16:29:20,570] Finished trial#114 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 11 order_s 115
profit -51270.0


[I 2019-06-27 16:29:30,250] Finished trial#115 resulted in value: 51270.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 2 eq 116
profit -7110.0


[I 2019-06-27 16:29:32,814] Finished trial#116 resulted in value: 7110.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 4 eq 117
profit 5710.0


[I 2019-06-27 16:29:37,095] Finished trial#117 resulted in value: -5710.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 3 eq 118
profit -103560.0


[I 2019-06-27 16:29:43,050] Finished trial#118 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 eq 119
profit -75990.0


[I 2019-06-27 16:29:49,829] Finished trial#119 resulted in value: 75990.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 120
profit -82750.0


[I 2019-06-27 16:30:07,861] Finished trial#120 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 7 eq 121
profit -93170.0


[I 2019-06-27 16:30:21,574] Finished trial#121 resulted in value: 93170.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 6 order_s 122
profit -55780.0


[I 2019-06-27 16:30:27,241] Finished trial#122 resulted in value: 55780.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 12 order_o 123
profit -38880.0


[I 2019-06-27 16:30:37,309] Finished trial#123 resulted in value: 38880.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 15 order_s 124
profit 38190.0


[I 2019-06-27 16:30:49,840] Finished trial#124 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 2 eq 125
profit -8400.0


[I 2019-06-27 16:30:51,573] Finished trial#125 resulted in value: 8400.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 13 eq 126
profit -79830.0


[I 2019-06-27 16:31:11,944] Finished trial#126 resulted in value: 79830.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 10 eq 127
profit -58290.0


[I 2019-06-27 16:31:25,976] Finished trial#127 resulted in value: 58290.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 11 eq 128
profit -49090.0


[I 2019-06-27 16:31:44,699] Finished trial#128 resulted in value: 49090.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 4 eq 129
profit 5710.0


[I 2019-06-27 16:31:49,254] Finished trial#129 resulted in value: -5710.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 16 order_s 130
profit -65080.0


[I 2019-06-27 16:32:03,478] Finished trial#130 resulted in value: 65080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 131
profit 97600.0


[I 2019-06-27 16:32:13,833] Finished trial#131 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 5 eq 132
profit -120840.0


[I 2019-06-27 16:32:32,321] Finished trial#132 resulted in value: 120840.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 9 order_s 133
profit -69960.0


[I 2019-06-27 16:32:42,307] Finished trial#133 resulted in value: 69960.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 134
profit -103560.0


[I 2019-06-27 16:32:48,542] Finished trial#134 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 8 eq 135
profit -90400.0


[I 2019-06-27 16:33:01,755] Finished trial#135 resulted in value: 90400.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 10 order_o 136
profit -88130.0


[I 2019-06-27 16:33:12,235] Finished trial#136 resulted in value: 88130.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 eq 137
profit 26850.0


[I 2019-06-27 16:33:21,900] Finished trial#137 resulted in value: -26850.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 5 eq 138
profit -98750.0


[I 2019-06-27 16:33:38,686] Finished trial#138 resulted in value: 98750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 7 eq 139
profit -70510.0


[I 2019-06-27 16:33:49,986] Finished trial#139 resulted in value: 70510.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 6 eq 140
profit -162640.0


[I 2019-06-27 16:34:08,580] Finished trial#140 resulted in value: 162640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 141
profit -104490.0


[I 2019-06-27 16:34:26,741] Finished trial#141 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 3 eq 142
profit -103560.0


[I 2019-06-27 16:34:32,756] Finished trial#142 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 143
profit 38190.0


[I 2019-06-27 16:34:44,379] Finished trial#143 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 9 order_s 144
profit -72930.0


[I 2019-06-27 16:34:52,613] Finished trial#144 resulted in value: 72930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_s 145
profit 82420.0


[I 2019-06-27 16:35:03,634] Finished trial#145 resulted in value: -82420.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_o 146
profit 19620.0


[I 2019-06-27 16:35:13,625] Finished trial#146 resulted in value: -19620.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 eq 147
profit 15620.0


[I 2019-06-27 16:35:23,944] Finished trial#147 resulted in value: -15620.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 148
profit 38190.0


[I 2019-06-27 16:35:35,721] Finished trial#148 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 4 eq 149
profit -93270.0


[I 2019-06-27 16:35:41,630] Finished trial#149 resulted in value: 93270.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 12 order_s 150
profit -51080.0


[I 2019-06-27 16:35:52,421] Finished trial#150 resulted in value: 51080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 2 eq 151
profit -51240.0


[I 2019-06-27 16:35:54,162] Finished trial#151 resulted in value: 51240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 3 eq 152
profit -103560.0


[I 2019-06-27 16:36:00,218] Finished trial#152 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 5 eq 153
profit -135240.0


[I 2019-06-27 16:36:19,406] Finished trial#153 resulted in value: 135240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_s 154
profit -66520.0


[I 2019-06-27 16:36:26,760] Finished trial#154 resulted in value: 66520.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 155
profit -82750.0


[I 2019-06-27 16:36:44,683] Finished trial#155 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 eq 156
profit 71430.0


[I 2019-06-27 16:36:52,427] Finished trial#156 resulted in value: -71430.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 5 eq 157
profit -80280.0


[I 2019-06-27 16:37:01,982] Finished trial#157 resulted in value: 80280.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 13 eq 158
profit -79830.0


[I 2019-06-27 16:37:20,795] Finished trial#158 resulted in value: 79830.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 7 eq 159
profit -88140.0


[I 2019-06-27 16:37:39,405] Finished trial#159 resulted in value: 88140.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 8 order_s 160
profit -52980.0


[I 2019-06-27 16:37:46,685] Finished trial#160 resulted in value: 52980.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 15 order_s 161
profit 38190.0


[I 2019-06-27 16:37:59,709] Finished trial#161 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 13 order_s 162
profit -32570.0


[I 2019-06-27 16:38:10,643] Finished trial#162 resulted in value: 32570.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 11 eq 163
profit -49090.0


[I 2019-06-27 16:38:28,734] Finished trial#163 resulted in value: 49090.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 7 eq 164
profit -93170.0


[I 2019-06-27 16:38:42,720] Finished trial#164 resulted in value: 93170.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 12 eq 165
profit -112220.0


[I 2019-06-27 16:38:59,991] Finished trial#165 resulted in value: 112220.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 14 order_s 166
profit -61120.0


[I 2019-06-27 16:39:13,724] Finished trial#166 resulted in value: 61120.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 10 order_s 167
profit -56930.0


[I 2019-06-27 16:39:22,985] Finished trial#167 resulted in value: 56930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 168
profit -103560.0


[I 2019-06-27 16:39:29,279] Finished trial#168 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 6 eq 169
profit -109530.0


[I 2019-06-27 16:39:44,055] Finished trial#169 resulted in value: 109530.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 4 eq 170
profit -62950.0


[I 2019-06-27 16:39:50,069] Finished trial#170 resulted in value: 62950.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 1 order_s 171
profit -54810.0


[I 2019-06-27 16:39:52,300] Finished trial#171 resulted in value: 54810.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 16 order_s 172
profit 57030.0


[I 2019-06-27 16:40:05,077] Finished trial#172 resulted in value: -57030.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 4 eq 173
profit -142020.0


[I 2019-06-27 16:40:13,057] Finished trial#173 resulted in value: 142020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 2 eq 174
profit -6750.0


[I 2019-06-27 16:40:15,710] Finished trial#174 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 15 order_s 175
profit 38190.0


[I 2019-06-27 16:40:28,072] Finished trial#175 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 5 eq 176
profit -90900.0


[I 2019-06-27 16:40:44,724] Finished trial#176 resulted in value: 90900.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 6 eq 177
profit -162640.0


[I 2019-06-27 16:41:03,249] Finished trial#177 resulted in value: 162640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 178
profit -104490.0


[I 2019-06-27 16:41:21,387] Finished trial#178 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 7 eq 179
profit -70510.0


[I 2019-06-27 16:41:33,317] Finished trial#179 resulted in value: 70510.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 180
profit 97600.0


[I 2019-06-27 16:41:43,237] Finished trial#180 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 6 eq 181
profit -102870.0


[I 2019-06-27 16:41:57,089] Finished trial#181 resulted in value: 102870.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 9 order_o 182
profit -76640.0


[I 2019-06-27 16:42:05,343] Finished trial#182 resulted in value: 76640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 12 order_s 183
profit -51080.0


[I 2019-06-27 16:42:16,025] Finished trial#183 resulted in value: 51080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 2 eq 184
profit -51240.0


[I 2019-06-27 16:42:17,753] Finished trial#184 resulted in value: 51240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 order_s 185
profit 81630.0


[I 2019-06-27 16:42:25,924] Finished trial#185 resulted in value: -81630.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 3 eq 186
profit -43680.0


[I 2019-06-27 16:42:32,001] Finished trial#186 resulted in value: 43680.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 4 eq 187
profit -148330.0


[I 2019-06-27 16:42:43,395] Finished trial#187 resulted in value: 148330.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 188
profit -82750.0


[I 2019-06-27 16:43:02,262] Finished trial#188 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 189
profit 97600.0


[I 2019-06-27 16:43:12,303] Finished trial#189 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_s 190
profit -66520.0


[I 2019-06-27 16:43:19,514] Finished trial#190 resulted in value: 66520.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 5 eq 191
profit -80280.0


[I 2019-06-27 16:43:28,969] Finished trial#191 resulted in value: 80280.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 8 eq 192
profit -67080.0


[I 2019-06-27 16:43:35,981] Finished trial#192 resulted in value: 67080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 12 eq 193
profit -68600.0


[I 2019-06-27 16:43:53,247] Finished trial#193 resulted in value: 68600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 7 eq 194
profit -93170.0


[I 2019-06-27 16:44:07,129] Finished trial#194 resulted in value: 93170.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 6 eq 195
profit -115790.0


[I 2019-06-27 16:44:20,740] Finished trial#195 resulted in value: 115790.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_o 196
profit 72290.0


[I 2019-06-27 16:44:31,655] Finished trial#196 resulted in value: -72290.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 14 order_s 197
profit -52160.0


[I 2019-06-27 16:44:43,302] Finished trial#197 resulted in value: 52160.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 11 eq 198
profit -49090.0


[I 2019-06-27 16:45:01,363] Finished trial#198 resulted in value: 49090.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 10 order_s 199
profit -56930.0


[I 2019-06-27 16:45:10,428] Finished trial#199 resulted in value: 56930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 12 eq 200
profit -112220.0


[I 2019-06-27 16:45:27,555] Finished trial#200 resulted in value: 112220.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 9 order_s 201
profit -69960.0


[I 2019-06-27 16:45:36,852] Finished trial#201 resulted in value: 69960.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 202
profit -103560.0


[I 2019-06-27 16:45:42,757] Finished trial#202 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 2 eq 203
profit -6750.0


[I 2019-06-27 16:45:45,263] Finished trial#203 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 eq 204
profit 15620.0


[I 2019-06-27 16:45:55,816] Finished trial#204 resulted in value: -15620.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 5 eq 205
profit -75630.0


[I 2019-06-27 16:46:04,540] Finished trial#205 resulted in value: 75630.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 8 eq 206
profit -90400.0


[I 2019-06-27 16:46:17,159] Finished trial#206 resulted in value: 90400.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 207
profit 38190.0


[I 2019-06-27 16:46:28,574] Finished trial#207 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 7 eq 208
profit -70510.0


[I 2019-06-27 16:46:40,429] Finished trial#208 resulted in value: 70510.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 4 eq 209
profit -46500.0


[I 2019-06-27 16:46:51,877] Finished trial#209 resulted in value: 46500.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 15 order_o 210
profit -82180.0


[I 2019-06-27 16:47:06,412] Finished trial#210 resulted in value: 82180.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 4 eq 211
profit -142020.0


[I 2019-06-27 16:47:14,291] Finished trial#211 resulted in value: 142020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 6 eq 212
profit -162640.0


[I 2019-06-27 16:47:32,450] Finished trial#212 resulted in value: 162640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 213
profit -104490.0


[I 2019-06-27 16:47:50,305] Finished trial#213 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 9 order_s 214
profit 77740.0


[I 2019-06-27 16:47:57,772] Finished trial#214 resulted in value: -77740.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 10 order_s 215
profit -58060.0


[I 2019-06-27 16:48:06,710] Finished trial#215 resulted in value: 58060.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 7 eq 216
profit -119300.0


[I 2019-06-27 16:48:26,237] Finished trial#216 resulted in value: 119300.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 2 eq 217
profit -51240.0


[I 2019-06-27 16:48:28,020] Finished trial#217 resulted in value: 51240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 12 order_s 218
profit -51080.0


[I 2019-06-27 16:48:38,797] Finished trial#218 resulted in value: 51080.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 eq 219
profit 15620.0


[I 2019-06-27 16:48:49,175] Finished trial#219 resulted in value: -15620.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 5 eq 220
profit -135240.0


[I 2019-06-27 16:49:07,367] Finished trial#220 resulted in value: 135240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 3 eq 221
profit -103560.0


[I 2019-06-27 16:49:13,314] Finished trial#221 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_o 222
profit -65930.0


[I 2019-06-27 16:49:20,816] Finished trial#222 resulted in value: 65930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 12 eq 223
profit -68600.0


[I 2019-06-27 16:49:37,942] Finished trial#223 resulted in value: 68600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 6 eq 224
profit -116250.0


[I 2019-06-27 16:49:55,931] Finished trial#224 resulted in value: 116250.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 225
profit -82750.0


[I 2019-06-27 16:50:13,779] Finished trial#225 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 8 order_s 226
profit -52980.0


[I 2019-06-27 16:50:21,121] Finished trial#226 resulted in value: 52980.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 7 eq 227
profit -88140.0


[I 2019-06-27 16:50:39,566] Finished trial#227 resulted in value: 88140.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 228
profit 38190.0


[I 2019-06-27 16:50:50,913] Finished trial#228 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 13 order_s 229
profit -32570.0


[I 2019-06-27 16:51:01,688] Finished trial#229 resulted in value: 32570.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 7 eq 230
profit -93170.0


[I 2019-06-27 16:51:15,725] Finished trial#230 resulted in value: 93170.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 12 eq 231
profit -112220.0


[I 2019-06-27 16:51:32,831] Finished trial#231 resulted in value: 112220.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 order_s 232
profit 81630.0


[I 2019-06-27 16:51:41,142] Finished trial#232 resulted in value: -81630.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 11 eq 233
profit -49090.0


[I 2019-06-27 16:52:00,160] Finished trial#233 resulted in value: 49090.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 16 eq 234
profit -72680.0


[I 2019-06-27 16:52:13,748] Finished trial#234 resulted in value: 72680.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 15 order_o 235
profit -67910.0


[I 2019-06-27 16:52:28,213] Finished trial#235 resulted in value: 67910.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 236
profit -103560.0


[I 2019-06-27 16:52:34,404] Finished trial#236 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 6 eq 237
profit -109530.0


[I 2019-06-27 16:52:49,235] Finished trial#237 resulted in value: 109530.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 8 eq 238
profit -90400.0


[I 2019-06-27 16:53:02,341] Finished trial#238 resulted in value: 90400.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 4 eq 239
profit -98810.0


[I 2019-06-27 16:53:13,663] Finished trial#239 resulted in value: 98810.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 14 order_s 240
profit -71000.0


[I 2019-06-27 16:53:28,011] Finished trial#240 resulted in value: 71000.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_s 241
profit 82420.0


[I 2019-06-27 16:53:38,964] Finished trial#241 resulted in value: -82420.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 5 eq 242
profit -90900.0


[I 2019-06-27 16:53:55,463] Finished trial#242 resulted in value: 90900.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 6 eq 243
profit -162640.0


[I 2019-06-27 16:54:13,853] Finished trial#243 resulted in value: 162640.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 order_s 244
profit 81630.0


[I 2019-06-27 16:54:22,235] Finished trial#244 resulted in value: -81630.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 2 eq 245
profit -6750.0


[I 2019-06-27 16:54:24,794] Finished trial#245 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 246
profit -104490.0


[I 2019-06-27 16:54:43,129] Finished trial#246 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 9 eq 247
profit -74580.0


[I 2019-06-27 16:54:50,943] Finished trial#247 resulted in value: 74580.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 4 eq 248
profit -142020.0


[I 2019-06-27 16:54:58,973] Finished trial#248 resulted in value: 142020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 7 eq 249
profit -119300.0


[I 2019-06-27 16:55:17,941] Finished trial#249 resulted in value: 119300.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 2 eq 250
profit -7110.0


[I 2019-06-27 16:55:20,552] Finished trial#250 resulted in value: 7110.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 251
profit 38190.0


[I 2019-06-27 16:55:32,042] Finished trial#251 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 13 order_s 252
profit -33220.0


[I 2019-06-27 16:55:43,761] Finished trial#252 resulted in value: 33220.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 5 eq 253
profit -83600.0


[I 2019-06-27 16:55:53,573] Finished trial#253 resulted in value: 83600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 3 eq 254
profit -103560.0


[I 2019-06-27 16:55:59,555] Finished trial#254 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 255
profit -82750.0


[I 2019-06-27 16:56:17,579] Finished trial#255 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_o 256
profit -65930.0


[I 2019-06-27 16:56:25,068] Finished trial#256 resulted in value: 65930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 257
profit 97600.0


[I 2019-06-27 16:56:35,195] Finished trial#257 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 258
profit 97600.0


[I 2019-06-27 16:56:45,317] Finished trial#258 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 259
profit 97600.0


[I 2019-06-27 16:56:55,579] Finished trial#259 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 eq 260
profit 26850.0


[I 2019-06-27 16:57:05,791] Finished trial#260 resulted in value: -26850.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 order_s 261
profit 81630.0


[I 2019-06-27 16:57:14,563] Finished trial#261 resulted in value: -81630.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_o 262
profit 72290.0


[I 2019-06-27 16:57:25,497] Finished trial#262 resulted in value: -72290.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 box 5 eq 263
profit -140220.0


[I 2019-06-27 16:57:34,906] Finished trial#263 resulted in value: 140220.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 6 eq 264
profit -116250.0


[I 2019-06-27 16:57:53,293] Finished trial#264 resulted in value: 116250.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t2 12 eq 265
profit -68600.0


[I 2019-06-27 16:58:10,673] Finished trial#265 resulted in value: 68600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 11 order_s 266
profit 108830.0


[I 2019-06-27 16:58:19,958] Finished trial#266 resulted in value: -108830.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow1 8 eq 267
profit -84790.0


[I 2019-06-27 16:58:39,746] Finished trial#267 resulted in value: 84790.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 4 order_s 268
profit -16960.0


[I 2019-06-27 16:58:43,657] Finished trial#268 resulted in value: 16960.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 flow2 9 order_s 269
profit -18470.0


[I 2019-06-27 16:58:51,258] Finished trial#269 resulted in value: 18470.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mflow 11 eq 270
profit -49090.0


[I 2019-06-27 16:59:09,638] Finished trial#270 resulted in value: 49090.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form22 11 eq 271
profit -92430.0


[I 2019-06-27 16:59:25,105] Finished trial#271 resulted in value: 92430.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 10 eq 272
profit 71430.0


[I 2019-06-27 16:59:32,672] Finished trial#272 resulted in value: -71430.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド flow 10 order_s 273
profit -56930.0


[I 2019-06-27 16:59:41,826] Finished trial#273 resulted in value: 56930.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


複勝 none 15 order_o 274
profit -67910.0


[I 2019-06-27 16:59:56,108] Finished trial#274 resulted in value: 67910.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_s 275
profit 82420.0


[I 2019-06-27 17:00:07,086] Finished trial#275 resulted in value: -82420.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 form21 6 eq 276
profit -109530.0


[I 2019-06-27 17:00:21,975] Finished trial#276 resulted in value: 109530.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow2 3 eq 277
profit -103560.0


[I 2019-06-27 17:00:28,336] Finished trial#277 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mbox 3 eq 278
profit -103560.0


[I 2019-06-27 17:00:34,581] Finished trial#278 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mbox 2 eq 279
profit -6750.0


[I 2019-06-27 17:00:37,337] Finished trial#279 resulted in value: 6750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 14 order_s 280
profit 38190.0


[I 2019-06-27 17:00:49,198] Finished trial#280 resulted in value: -38190.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


単勝 none 9 order_s 281
profit -90060.0


[I 2019-06-27 17:00:58,703] Finished trial#281 resulted in value: 90060.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 form2 8 eq 282
profit -90400.0


[I 2019-06-27 17:01:12,054] Finished trial#282 resulted in value: 90400.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 mflow 7 eq 283
profit -70510.0


[I 2019-06-27 17:01:23,372] Finished trial#283 resulted in value: 70510.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 form2t1 4 eq 284
profit -142020.0


[I 2019-06-27 17:01:31,289] Finished trial#284 resulted in value: 142020.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow1 5 eq 285
profit -139070.0


[I 2019-06-27 17:01:42,794] Finished trial#285 resulted in value: 139070.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 286
profit 97600.0


[I 2019-06-27 17:01:52,999] Finished trial#286 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 form2 11 eq 287
profit -104490.0


[I 2019-06-27 17:02:11,505] Finished trial#287 resulted in value: 104490.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 box 7 eq 288
profit -119300.0


[I 2019-06-27 17:02:31,727] Finished trial#288 resulted in value: 119300.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 flow 10 eq 289
profit 30520.0


[I 2019-06-27 17:02:40,580] Finished trial#289 resulted in value: -30520.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


普通馬複 flow 10 order_o 290
profit -83760.0


[I 2019-06-27 17:02:49,858] Finished trial#290 resulted in value: 83760.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 12 order_s 291
profit 97600.0


[I 2019-06-27 17:03:00,039] Finished trial#291 resulted in value: -97600.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド box 2 eq 292
profit -51240.0


[I 2019-06-27 17:03:01,852] Finished trial#292 resulted in value: 51240.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


馬単 mbox 4 eq 293
profit -103690.0


[I 2019-06-27 17:03:13,435] Finished trial#293 resulted in value: 103690.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 mflow1 3 eq 294
profit -103560.0


[I 2019-06-27 17:03:19,466] Finished trial#294 resulted in value: 103560.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


ワイド form2 11 eq 295
profit -82750.0


[I 2019-06-27 17:03:37,753] Finished trial#295 resulted in value: 82750.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連単 tflow2 13 order_s 296
profit 82420.0


[I 2019-06-27 17:03:49,288] Finished trial#296 resulted in value: -82420.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠複 flow 8 order_s 297
profit -66520.0


[I 2019-06-27 17:03:56,877] Finished trial#297 resulted in value: 66520.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


枠単 flow 6 order_s 298
profit -55780.0


[I 2019-06-27 17:04:02,698] Finished trial#298 resulted in value: 55780.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


三連複 box 5 eq 299
profit -80280.0


[I 2019-06-27 17:04:12,303] Finished trial#299 resulted in value: 80280.0. Current best value is -108830.0 with parameters: {'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}.


In [79]:
study.best_params

{'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}

In [80]:
UMAKAZU

16

# 結果

### 6頭

```
{'shiki': '三連単-tflow2', 'pattern': 'eq', 'select_num': 4}
```
対象レース: 6

### 7頭

+190円

```
{'shiki': '馬単-mbox', 'select_num': 2}
```
対象レース: 16

### 8頭

-300円

```
{'shiki': '単勝-none', 'pattern': 'eq', 'select_num': 7}
```
対象レース: 32

### 9頭

+11,970円

```
{'shiki': '三連複-flow1', 'select_num': 3}
```
対象レース: 69

### 10頭

+35,980円

```
{'shiki': '三連単-form2t2', 'select_num': 10}
```
対象レース: 99

### 11頭

+307,560円

```
{'shiki': '三連単-mbox', 'select_num': 3}
```

対象レース: 198

### 12頭

+26,910円

```
{'shiki': '三連単-form2t2', 'select_num': 8}
```

対象レース: 503

### 13頭

+170,910円

```
{'shiki': '馬単-flow', 'pattern': 'eq', 'select_num': 11}
```

対象レース: 85

### 14頭

+218,930円

```
{'shiki': '三連複-form22', 'select_num': 5}
```

対象レース: 202

### 15頭

+53,240円

```
{'shiki': '馬単-flow', 'pattern': 'eq', 'select_num': 11}
```
対象レース: 70


### 16頭

+108,830円

```
{'shiki': '三連単-tflow2', 'pattern': 'order_s', 'select_num': 11}
```
対象レース: 147
